frontend/src/console/components/ConsolePage/displayMessages.ts

```diff
@@ -2,6 +2,11 @@ const messages = {
   editCourses: {
     defaultMessage: 'Edit courses (Studio)',
     description: 'Label to Studio link.'
+  },
+  verifyEmail: {
+    defaultMessage:
+      'Some of your email addresses are pending confirmation. Please confirm them to begin managing your Open edX instance.',
```

Comment:
@xitij2000, @rusrushal13 (CC @giovannicimolin), [this comment](https://github.com/open-craft/opencraft/pull/581#discussion_r431072778) is relevant here. Posted there by mistake.